Incorrect example ?

Hello !

Thanks for your work !
I am trying to use your extension but I encountered some issues, with your example I have an error:
```
Caused by: io.quarkiverse.openfga.client.model.FGAValidationException: Invalid tuple 'thing:1#owner@1'. Reason: the 'user' field must be an object (e.g. document:1) or an 'object#relation' or a typed wildcard (e.g. group:*)
```

by chaning this [line](https://github.com/quarkiverse/quarkus-zanzibar/blob/main/docs/modules/ROOT/examples/ThingResource.java#L33) and  after reading the documentation and replacing `principal.getName()` with `"user:"+principal.getName()` It works.

Is there something I'm missing ?

--- a/docs/modules/ROOT/examples/ThingResource.java
+++ b/docs/modules/ROOT/examples/ThingResource.java
@@ -30,7 +30,7 @@ class ThingResource {
         return thingRepository.createThing(name)
                 .flatMap((thing) -> {
                     // <4>
-                    var relationship = new Relationship("thing", thing.getId(), "owner", principal.getName());
+                    var relationship = new Relationship("thing", thing.getId(), "owner", "user:" + principal.getName());
                     // <5>
                     return relationshipManager.add(List.of(relationship))
                             .map((unused) -> thing);

